### DISCUSSION TITLE
Is the a way to clear the input value ?

### DISCUSSION THREAD

**Discussion by invaders-xx** [RVxLab/filament-colorpicker#18] [Q&A] [answered]
Is there a way to set the field to null (clear the input value) ?

**Comment by RVxLab** [comment]
Not at this time. I attempted to add this earlier but I couldn't find a good way to integrate this with Vanilla Picker.

If you have any ideas I'd love to hear it.

**Comment by invaders-xx** [comment, marked as answer]
I used this package which works very well and has a clear button:
https://github.com/Simonwep/pickr

Let me know if you'd consider updating your package with this one, otherwise I will create a new one

**Reply by RVxLab** [reply]
While that picker has a clear button it does work very similarly to vanilla picker so I don’t see the benefit of switching. 

Since I do want to add this functionality I’ll have to come up with a different way. 

I’m gonna think about this, but I think it will be a separate clear button on the input.

**Reply by invaders-xx** [reply]
I am ok with you ;-) Thanks for your feedback.

**Reply by invaders-xx** [reply]
Do you have any idea when you will be able to add this functionality ?

**Reply by RVxLab** [reply]
I’ll try to add something towards the weekend.

**Reply by RVxLab** [reply]
This has been added and is available in 1.4
